Gopkg.toml

```diff
@@ -144,6 +144,10 @@ required = [
   name = "github.com/docker/distribution"
   version = "v2.7.1"
 
+[[constraint]]
+  branch = "master"
```

Comment:
This should bind to a release or a commit to ensure that eventual breaking changes will not break the machine-controller

Reply:
Fixed